docs/set-up.asciidoc

```diff
@@ -6,34 +6,90 @@ you must first {apm-server-ref-v}/configuration-rum.html[enable the RUM endpoint
 
 Once the APM Server endpoint has been configured, you can:
 
-* <<install-the-agent>>
+* <<usage>>
 * <<configuring-cors>>
 
-[[install-the-agent]]
-=== Install the Agent
+[[usage]]
+=== Usage
 
-There are three ways to use the APM RUM Agent:
+There are three ways to include the APM RUM Agent:
+
+APM RUM Agent can be added to your web page in one of two ways
 
-* <<using-package-managers>>: Install the Agent as a dependency to your application.
 * <<using-script-tags>>: Add a `<script>` tag to the HTML page.
-* <<using-production-build>>: Optimized bundles.
+* <<using-bundlers>>: Bundling the Agent package during the application's build process
 
 [float]
-[[using-package-managers]]
-=== Using Package Managers
+[[using-script-tags]]
+=== Using Script Tags
+
+==== Synchronous / Blocking Pattern
+
+Add a <script> tag to load the bundle and use the `elasticApm` global
+object to initialize the agent:
+
+[source,html]
+----
+<script src="https://<your-cdn-host>.com/path/to/elastic-apm-rum.umd.min-<version>.js" crossorigin></script>
+<script>
+  elasticApm.init({
+    serviceName: '<instrumeted-app>',
+    serverUrl: '<apm-server-url>',
+  })
+</script>
+----
+
+==== Asynchronous / Non-Blocking Pattern
+
+Loading the script asynchronously ensures the agent script will not block other
+resources on the page, however, it will stick block browsers `onload` event.
+
+[source,html]
+----
+<script>
+  ;(function(d, s, c) {
+    var j = d.createElement(s),
+      t = d.getElementsByTagName(s)[0]
+
+    j.src = 'https://<your-cdn-host>.com/path/to/elastic-apm-rum.umd.min-<version>.js'
+    j.onload = function() {elasticApm.init(c)}
+    t.parentNode.insertBefore(j, t)
+  })(document, 'script', {serviceName: '<instrumeted-app>', serverUrl: '<apm-server-url>'})
+</script>
+----
+
+Even though this is the recommended pattern, there is a caveat
+attached to it as a result of downloading and initializing the
+agent asynchronously. The distributed tracing would not work for requests that
+happened before the agent is initialized as the agent would not be available to
+patch the requests on time.
```

Comment:
Maybe something like this? It simplifies it a bit, but hopefully not too much:

```
Even though this is the recommended pattern, there is a caveat to be aware of.
Because the downloading and initializing of the agent happens asynchronously,
distributed tracing will not work for requests that occur before the agent is initialized.
```